src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -421,16 +416,8 @@
   }
 }
 
-.pf-c-dropdown__group {
-  padding-top: var(--pf-c-dropdown__group--PaddingTop); // remove at breaking change
-
-  &:first-child {
-    --pf-c-dropdown__group--PaddingTop: var(--pf-c-dropdown__group--first-child--PaddingTop); // remove at breaking change
-  }
-
-  & + & {
-    --pf-c-dropdown__group--PaddingTop: var(--pf-c-dropdown__group--group--PaddingTop);
-  }
+.pf-c-dropdown__group + .pf-c-dropdown__group {
+  padding-top: var(--pf-c-dropdown__group--group--PaddingTop);
```

Comment:
Should this be updated to `--pf-c-dropdown__group__group--PaddingTop`?

Reply:
I'd say no since it's `__group + __group`, not `__group __group`